src/tr1/game/objects/creatures/bear.c

```diff
@@ -76,6 +71,7 @@ static void M_Setup(OBJECT *const obj)
 
 static void M_Control(const int16_t item_num)
 {
+    Object_Get(O_BEAR)->pivot_length = g_Config.gameplay.fix_bear_ai ? 0 : 500;
```

Comment:
Let's avoid assignments to a pointer from a function call, eg let's introduce an `obj` variable on which we later execute state-altering operations.